woocommerce/woocommerce
# Add the "Channels" card into Marketing page

### All Submissions:


### Changes proposed in this Pull Request:

Part of https://github.com/woocommerce/woocommerce/issues/34906.

This is a PR to add the "Channels" card into the Marketing page.

Note: This draft PR currently uses some React hooks with dummy data to render the UI. The hooks will be changed to call APIs when APIs are developed.

Screenshot:

![image](https://user-images.githubusercontent.com/417342/207595325-c67ee8ff-718b-4d4a-b2c8-73adb03b6c7c.png)

### How to test the changes in this Pull Request:



Pre-conditions: Enable WooCommerce Multichannel Marketing experience in WooCommerce Settings > Advanced > Features page: `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`

1. Go to WooCommerce Marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`
2. You should see the Channels card like the screenshot above.

### Other information:


### FOR PR REVIEWER ONLY:
